src/HomeWorkApp_1.java

```diff
@@ -0,0 +1,57 @@
+public class HomeWorkApp_1 {
+    public static void main(String[] args) {
+//      printThreeWords();
+//      checkSumSign();
+//      printColor();
+//      compareNumbers();
+
+    }
+
+    public static void printThreeWords(){
+        System.out.println("Orange");
+        System.out.println("Banana");
+        System.out.println("Apple");
+    }
+
+    public static void checkSumSign(){
+        int a;
+        int b;
+        int sum;
+
+        a = 5;
+        b = -8;
+        sum = a + b;
+
+        if(sum >= 0){
+            System.out.println("Сумма положительная");
+        }
+        else {
+            System.out.println("Сумма отрицательная");
+        }
+    }
+    public static void printColor(){
+        int value;
+
+        value = 0;
+        if (value <= 0){
+            System.out.println("Красный");
+        }
+        else if (value > 0 && value <= 100 ){
```

Comment:
```suggestion
        else if (value <= 100)
```
поскольку данное условие работает само по себе из-за неактульности первого `value<= 0 -> false <=> value > 0`